sdk/tools/gen_baseaddress.py

```diff
@@ -253,7 +255,10 @@ def gen_baseaddress(self, output_file):
             postfix = ' # should be above 0x%08x' % self.address
         elif self._reserved:
             postfix = ' # reserved'
-        output_file.write('set(baseaddress_%-30s 0x%08x)%s\n' % (name, self.address, postfix))
+        # Current longest name is: 'msvcrt_crt_dll_startup' (22).
+        if len(name) > 22:
+            print('#', name, 'is longer than current width:', len(name), '> 22')
+        output_file.write('set(baseaddress_%-22s 0x%08x)%s\n' % (name, self.address, postfix))
```

Comment:
What (problem) solves (this) ?
Are you trying to view this file on a monitor with a resolution of 400x300 ?

Reply:
Trivial space optimization.
Indeed, easier to read with smaller blank in the middle...

Reply:
These are not meant for humans to read them, and even if they were this complicates the code without any added benefit (in my opinion).